.eslintrc.json

```diff
@@ -51,8 +51,7 @@
             "ul": ["list"],
             "ol": ["list"]
           }
-        ],
-        "no-console": ["error", { "allow": ["warn", "error"] }]
+        ]
```

Comment:
@mkjmdski  this is not for merging right? 

I will leave this conversation open here.